proposals/0004-register-types-and-diagnostics.md

```diff
@@ -59,7 +59,10 @@ compatible with the Decl. All of this analysis and validation will be executed
 inside a new function, `DiagnoseHLSLRegisterAttribute`. This function will be
 responsible for validating the semantic meaning behind the application of the
 attribute, while the rest of `handleResourceBindingAttr` is responsible for
-validating the syntax of the attribute.
+validating the syntax of the attribute. 
```

Comment:
nit: did you intend the stuff about space to be on a new paragraph? If so, you need an extra blank line.